docs/sphinx/BUILD.bazel

```diff
@@ -112,6 +122,8 @@ sphinx_build_binary(
         requirement("sphinx_rtd_theme"),
         requirement("myst_parser"),
         requirement("readthedocs_sphinx_ext"),
+        requirement("typing_extensions"),
+        "//sphinxdocs/src/sphinx_stardoc",
```

Comment:
I thought that `shpinx_stardoc` includes the `typing_extensions`? Is it really needed here as well?

Reply:
It used to, but I removed it from the sphinx_stardoc target so it could be consumed by users outside our repo. There was some interest from others in using this sphinx and stardoc stuff, so I figured that'd give them at least some way to try it out in the meantime.